### PR TITLE
[IMPROVE] Migrate authorization logic into Rocket.Chat side

### PR DESCRIPTION
**Pull request by shiqimei**
Related commits: https://github.com/lolimay/Button-War-rocketchat-demo/commit/aaee8774224a588a38246c4efdd567a2c1361c33

### CODE REVIEW

File: app/koji-demo/client/index.js [hunk 1]
```diff
@@ -42,19 +43,31 @@ Meteor.startup(function() {
...
 			if (action === 'getUserInfo') {
-				source.postMessage({
-					rcEmbeddedSdk: {
-						version: '0.0.1',
-						action: 'connected',
-						connected: {
-							username,
-							avatarUrl,
-							roomName,
+				const authorized = await authorize(appName);
```
Comment (kb0304) [inline app/koji-demo/client/index.js]: Can we rename it to something like `authorizeMiniApp`?

File: app/ui-utils/client/index.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export { menu } from './lib/menu';
 export { call } from './lib/callMethod';
 export { erase, hide, leave } from './lib/ChannelActions';
 export { MessageAction } from './lib/MessageAction';
+export { miniAppAtion } from './lib/miniAppAction';
```
Comment (kb0304) [inline app/ui-utils/client/index.js]: `miniAppAction` instead of `miniAppAtion ` ?

Comment (kb0304) [reply app/ui-utils/client/index.js]: Where is it being used??

**Review by kb0304 — CHANGES_REQUESTED** [review]

File: app/ui-utils/client/index.js (hunk 2 above)
Comment (shiqimei) [reply app/ui-utils/client/index.js]: This is not used yet. I just expose it the index file to let others know this lib. Should we remove it?

Comment (kb0304) [reply app/ui-utils/client/index.js]: We can keep it, just fix the typo

File: app/koji-demo/client/index.js [hunk 3]
```diff
@@ -42,19 +43,31 @@ Meteor.startup(function() {
...
-				source.postMessage({
-					rcEmbeddedSdk: {
-						version: '0.0.1',
-						action: 'connected',
-						connected: {
-							username,
-							avatarUrl,
-							roomName,
+				const authorized = await authorizeMiniApp(appName);
+				console.log(authorized);
```
Comment (kb0304) [inline app/koji-demo/client/index.js]: Remove the log statement?

Comment (kb0304) [inline app/koji-demo/client/index.js:50]: Remove the `log` statement?

**Pull request closed without merging**